backend/Models/Data/Sighting.cs

```diff
@@ -0,0 +1,20 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+
+namespace WhaleSpotting.Models.Data;
+
+public class Sighting
+{
+    [Key]
+    public int Id { get; set; }
+    public int UserId { get; set; }
+    public int WhaleSpeciesId { get; set; }
+    public float Lattitude { get; set; }
+    public float Longitude { get; set; }
+    public string PhotoUrl { get; set; }
+    public string? Description { get; set; }
+    public DateTime DateTime { get; set; }
+    public bool IsApproved { get; set; }
```

Comment:
nice - can you also make this default to false? let me know if you have any questions on how to do that